src/scripts/main — копия.js

```diff
@@ -0,0 +1,253 @@
+'use strict';
+
+let gameField;
+let score = 0;
+let change;
+
+const button = document.querySelector('button');
+const startMessages = document.querySelector('.message-start');
+const loseMessages = document.querySelector('.message-lose');
+const winMessages = document.querySelector('.message-win');
+const elementScore = document.querySelector('.game-score');
+
+// function createField() {
+//   return [
+//     [0, 0, 0, 0],
+//     [0, 0, 0, 0],
+//     [0, 0, 0, 0],
+//     [0, 0, 0, 0],
+//   ];
+// }
+
+function fillGameField(board) {
+  const cells = document.querySelectorAll('.field-cell');
+  let count = 0;
+
+  board.flat().forEach((item, i) => {
+    if (item) {
+      cells[i].textContent = item;
+      cells[i].className = `field-cell field-cell--${item}`;
+
+      if (item === 2048) {
+        document.removeEventListener('keydown', keyPress);
+        winMessages.classList.remove('hidden');
+      }
+
+      count++;
+    } else {
+      cells[i].textContent = '';
+      cells[i].className = 'field-cell';
+    }
+  });
+
+  let stopGame = true;
+
+  board.forEach((element, i) => {
+    element.forEach((item, j) => {
+      if (item === board[i][j + 1]) {
+        stopGame = false;
+      }
+
+      if (i < 3 && item === board[i + 1][j]) {
+        stopGame = false;
+      }
+    });
+  });
+
+  // for (let i = 0; i < 4; i++) {
+  //   for (let j = 0; j < 3; j++) {
+  //     if (board[i][j] === board[i][j + 1]) {
+  //       stopGame = false;
+  //     }
+
+  //     if (i < 3 && board[i][j] === board[i + 1][j]) {
+  //       stopGame = false;
+  //     }
+  //   }
+  // }
```

Comment:
Don't leave any commented code. Just remove it everywhere

Reply:
it was a copy that was accidentally added, I deleted it